modules/openapi-generator/src/main/resources/python/python-experimental/model_utils.mustache

```diff
@@ -724,14 +724,14 @@ def deserialize_primitive(data, klass, path_to_item):
                     # '7' -> 7.0 -> '7.0' != '7'
                     raise ValueError('This is not a float')
             return converted_value
-    except (OverflowError, ValueError):
+    except (OverflowError, ValueError) as ex:
         # parse can raise OverflowError
         raise ApiValueError(
             "{0}Failed to parse {1} as {2}".format(
                 additional_message, repr(data), get_py3_class_name(klass)
             ),
             path_to_item=path_to_item
-        )
+        ) from ex
```

Comment:
@spacether , any good idea on how to enable this for python 3.x without breaking 2.x?